label_studio/core/settings/base.py

```diff
@@ -310,8 +310,8 @@
 os.makedirs(EXPORT_DIR, exist_ok=True)
 
 # file / task size limits
-DATA_UPLOAD_MAX_MEMORY_SIZE = int(get_env('DATA_UPLOAD_MAX_MEMORY_SIZE', 50 * 1024 * 1024))
-TASKS_MAX_NUMBER = 250000
+DATA_UPLOAD_MAX_MEMORY_SIZE = int(get_env('DATA_UPLOAD_MAX_MEMORY_SIZE', 250 * 1024 * 1024))
+TASKS_MAX_NUMBER = 1000000
```

Comment:
I don't think we need to do this by default. It's better to wrap it into `TASKS_MAX_NUMER=get_env("TASKS_MAX_NUMBER", default=250000)` then writing the message about increasing this limit (with "not recommended") right after the corresponding error pops up

Reply:
A lot of people flare facing this issue. But I thought about get_env too

Reply:
still not sure if we enable this - could we run into another performance issue? I would shrink that now until performance enhancements